pyteal/ast/acct.py

```diff
@@ -55,3 +57,33 @@ def authAddr(cls, acct: Expr) -> MaybeValue:
 
 
 AccountParam.__module__ = "pyteal"
+
+
+class AccountParamObject:
+    """Represents information about an account"""
+
+    def __init__(self, account: Expr) -> None:
+        """Create a new AccountParamObject for the given account.
+
+        Args:
+            app: An index into Txn.accounts that corresponds to the application to check or an
```

Comment:
```suggestion
            account: An index into Txn.accounts that corresponds to the application to check or an
```
Not sure if this should be changed or the parameter should be renamed to accomodate the comment

Reply:
Reading more of the PR, it seems like this is just a copy/paste error and `account` makes sense here.